src/TagBoxContainer.jsx

```diff
@@ -116,6 +118,7 @@ export default class TagBoxContainer extends Component {
           tags={this.tags()}
           select={(t) => this.select(t)}
           create={() => this.createTag()}
+          search={(t, input, selectedTags) => search(t, input, selectedTags)}
```

Comment:
I suspect that:

```javascript
search={search}
```

will work in this case. The others are anonymous functions because they reference `this`.